src/content/en/updates/2017/09/autoplay-policy-changes.md

```diff
@@ -92,16 +92,23 @@ permission, it can delegate that permission to cross-origin iframes with a new
 same-origin iframes.
 
 <pre class="prettyprint">
-&lt;iframe src="myvideo.html" allow="autoplay">
+&lt;!-- Autoplay is allowed. -->
+&lt;iframe src="https://cross-origin.com/myvideo.html" allow="autoplay">
+
+&lt;!-- Autoplay and Fullscreen are allowed. -->
+&lt;iframe src="https://cross-origin.com/myvideo.html" allow="autoplay; fullscreen">
 </pre>
 
 When the feature policy for autoplay is disabled, calls to `play()` without a
 user gesture will reject the promise with a `NotAllowedError` DOMException. And
 the autoplay attribute will also be ignored.
 
-Note: You can try out this feature policy by enabling the experimental flag
+You can try out this feature policy by enabling the experimental flag
 `chrome://flags/#enable-experimental-web-platform-features`.
 
+Warning: Contrary to what was announced previously, `<iframe gesture=media>` is
```

Comment:
We never said <iframe gesture=media> was supported. Instead:

Warning: Older articles incorrectly recommend using the attribute _gesture=media_ which is not supported.